### PR TITLE
[MAINTENANCE] add the internal `GE_DATA_CONTEXT_ID` env var to the docker file

### PR DESCRIPTION
**Pull request by Kilo59**
Changes proposed in this pull request:
- add a known uuid as the `GE_DATA_CONTEXT_ID` so that the analytics team can filter out usage stats from CI builds and dev environments

### CODE REVIEW

**Review by tannerbeam — APPROVED** [review]

File: docker/Dockerfile.tests [hunk 1]
```diff
@@ -29,3 +29,6 @@ RUN pip install --no-cache-dir --requirement requirements-dev-test.txt --constra
 
 FROM dev as dev-tools
 RUN pip install --no-cache-dir --requirement requirements-dev-tools.txt --constraint constraints-dev.txt
+
+# used to filter usage stats coming from CI and dev environments
```
Comment (billdirks) [inline docker/Dockerfile.tests]: Let's move this to line 23, right after `FROM build_${SOURCE} AS dev`. This means the `dev` target and any derived targets (`test` and `dev-tools`) will have this set. The `test` target is what the azure pipelines use.

I've verified this works by building `test` after this change and seeing the env variable being set:
```
# echo ${GE_DATA_CONTEXT_ID}
00000000-0000-0000-0000-00000000e009
```

**Review by billdirks — CHANGES_REQUESTED** [review]
Thanks for making this change! We definitely don't want to be analyzing the sessions for the azure pipelines.

I'd move this env variable higher in the dockerfile.

File: docker/Dockerfile.tests (hunk 1 above)
Comment (Kilo59) [reply docker/Dockerfile.tests]: Done

**Review by billdirks — APPROVED** [review]
LGTM!

**Pull request merged**